circuits/modulus.circom

```diff
@@ -338,84 +338,84 @@ template ModulusAgainst2Q() {
 
 template ModulusWith25519Chunked51(n) {
```

Comment:
Here also we can pass a base and then make rest of the constants dependent on that base.